build-livecd-root

```diff
@@ -35,7 +35,7 @@ cd $tmpdir
 echo Working in directory $tmpdir
 
 echo "* Running livecd-creator"
-livecd-creator -v --title="Discovery Image" --compression-type=xz --cache /var/cache/build-fdi --config $srcdir/fdi-image.ks -f fdi -t /tmp
+livecd-creator -v --product="Discovery Image" --title="Foreman" --compression-type=xz --cache /var/cache/build-fdi --config $srcdir/fdi-image.ks -f fdi -t /tmp
```

Comment:
This just changes the title, it was "Fdi" which is not nice. Shown on the SYSLINUX screen (one second countdown) or Grub2.